integrations/discord/events.go

```diff
@@ -7,19 +7,51 @@ import (
 	"go.uber.org/zap"
 
 	"go.autokitteh.dev/autokitteh/integrations/discord/internal/vars"
+	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
 )
 
 func (h *handler) handleEvent(event any, eventType string) {
-	akEvent, err := h.transformEvent(event, eventType)
-	if err != nil {
+	var authorID string
+
+	// TODO(ENG-1546) - Add support for more event types
+	switch e := event.(type) {
+	case *discordgo.MessageCreate:
+		authorID = e.Author.ID
+	case *discordgo.MessageUpdate:
+		authorID = e.Author.ID
+	case *discordgo.MessageDelete:
+		authorID = e.Author.ID
+	default:
+		h.logger.Error("Unsupported event type", zap.String("eventType", eventType))
```

Comment:
Please initialize at the top of this function `l := h.logger.With(...)` the event type, and use this `l` instead of `h.logger`. In other words, the event type should be added to all log messages here.